src/data/graphql/typeDefs.ts

```diff
@@ -62,6 +62,7 @@ export default gql`
     domainPurpose: String!
     domainColor: String!
     blockNumber: Int!
+    motionState: String
```

Comment:
Shouldn't this be `String!` ? I mean, the motion will always have some state

Reply:
unless it wasn't a motion?